src/fragments/Message.tsx

```diff
@@ -3,13 +3,18 @@ import invariant from 'invariant';
 import React from 'react';
 import { TextMessage } from 'react-chat-ui-kit';
 
+import LinkPreview from './LinkPreview';
+
 type MessageProps = {
   message: ChatKittyMessage;
 };
 
 const Message: React.FC<MessageProps> = ({ message }: MessageProps) => {
   if (isTextMessage(message)) {
-    return <TextMessage text={message.body} />;
+    return (<div>
```

Comment:
is this unstyled div needed? or could it be a fragment instead. It's not ideal to clutter the DOM.

Reply:
replaced the div with the empty tag